nucypher/blockchain/eth/utils.py

```diff
@@ -162,78 +160,22 @@ def get_default_rpc_endpoints() -> Dict[int, List[str]]:
         return {}
 
 
-def get_healthy_default_rpc_endpoints(chain_id: int) -> List[str]:
+def get_healthy_default_rpc_endpoints(domain: str) -> Dict[int, List[str]]:
```

Comment:
Let's use the domain enum here. It makes checking domain a lot easier and less hardcoded.
```suggestion
def get_healthy_default_rpc_endpoints(domain: TACoDomain) -> Dict[int, List[str]]:
```

Reply:
done